python-avd/pyavd/templater.py

```diff
@@ -57,9 +57,9 @@ def import_filters_and_tests(self) -> None:
         from .vendor.j2.filter.hide_passwords import hide_passwords
         from .vendor.j2.filter.list_compress import list_compress
         from .vendor.j2.filter.natural_sort import natural_sort
+        from .vendor.j2.filter.range_expand import range_expand
```

Comment:
@ClausHolbechArista - When I reverted the commit there was a merge conflict on this file which I resolved, but please double-check I did this correctly!

Reply:
Looks good.